packages/cli/src/commands/typescript/generate/handleGenerate.mts

```diff
@@ -124,34 +128,68 @@ async function generateClientSdk(
   ontology: WireOntologyDefinition,
   args: TypescriptGenerateArgs,
 ) {
-  const minimalFs = {
-    writeFile: (path: string, contents: string) => {
-      return fs.promises.writeFile(path, contents, "utf-8");
-    },
-    mkdir: async (path: string, options?: { recursive: boolean }) => {
-      await fs.promises.mkdir(path, options);
-    },
-    readdir: async (path: string) => fs.promises.readdir(path),
-  };
-
+  const minimalFs = createNormalFs();
   try {
-    if (args.beta) {
-      await generateClientSdkVersionTwoPointZero(
+    if (args.clean) {
+      consola.info("Cleaning output directory", args.outDir);
+      await fs.promises.rm(args.outDir, { recursive: true, force: true });
+    }
+
+    if (!args.asPackage) {
+      await generateSourceFiles(args, ontology, createNormalFs());
+      return true;
+    }
+
+    const packageName = args.packageName;
+    if (!packageName) throw new Error("Package name is require");
+    const version = args.version;
```

Comment:
Is version not required? If it is, should we do an error check here as well?

Reply:
version is required already in the yargs